dev/docker/compose-v3

```diff
@@ -3,6 +3,5 @@ set -eou pipefail
 
 docker compose \
   -f dev/docker/docker-compose.yml \
-  --env-file dev/docker/local.env \
```

Comment:
What happened to these env vars?

Reply:
Martin flagged that they were d14n related but i merged the PR too fast: https://github.com/xmtp/libxmtp/pull/1837/files/f0891a6d03ad49429694f9b5f678fdcdef4b9b68#diff-5d75c69056c7813dd7e4244ca4445b5586ecd37c9ec61612bce308d3cf3f171a